components/engine_rocks/src/raft_engine.rs

```diff
@@ -221,6 +221,13 @@ impl RaftEngine for RocksEngine {
     fn dump_stats(&self) -> Result<String> {
         MiscExt::dump_stats(self)
     }
+
+    fn get_engine_size(&self) -> Result<u64> {
+        let handle = util::get_cf_handle(self.as_inner(), CF_DEFAULT)?;
```

Comment:
Why not checking other cf? or maybe we can reuse `get_engine_used_size`.

Reply:
`get_engine_used_size` is for kvdb instead of raftdb. Here `get_engine_size` is a trait method of `RaftEngine`, so only `CF_DEFAULT` needs to be considered.